notebook/services/eventlog/handlers.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from tornado import web
+
+from notebook.utils import url_path_join
+from notebook.base.handlers import APIHandler, json_errors
+from jupyter_telemetry.eventlog import EventLog
+
+class EventLoggingHandler(APIHandler):
+    """
+    A handler that receives and stores telemetry data from the client.
+    """
+    @json_errors
+    @web.authenticated
+    def post(self, *args, **kwargs):
+        try:
+            # Parse the data from the request body
+            raw_event = json.loads(self.request.body.strip().decode())
+        except Exception as e:
+            raise web.HTTPError(400, str(e))
```

Comment:
We're doing str(e) here and just passing e in line 35 - Lets keep it consistent?